metadata/src/main/java/org/apache/kafka/controller/ReplicationControlManager.java

```diff
@@ -321,22 +339,17 @@ public void replay(PartitionRecord record) {
         }
         PartitionControlInfo newPartInfo = new PartitionControlInfo(record);
         PartitionControlInfo prevPartInfo = topicInfo.parts.get(record.partitionId());
+        String topicPart = topicInfo.name + "-" + record.partitionId();
         if (prevPartInfo == null) {
-            log.info("Created partition {}:{} with {}.", record.topicId(),
-                record.partitionId(), newPartInfo.toString());
+            log.info("Created partition {} with {}.", topicPart, newPartInfo.toString());
```

Comment:
Is it intentional to use "name" to replace "id"?

Reply:
Yes, this was an intentional change to make it clearer which topic was getting created.  The ID is still available in the metadata log, of course.